v2/googlecloud-to-googlecloud/src/test/java/com/google/cloud/teleport/v2/templates/bigtablechangestreamstobigquery/BigtableChangeStreamsToBQIT.java

```diff
@@ -0,0 +1,520 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.templates.bigtablechangestreamstobigquery;
+
+import static com.google.cloud.teleport.it.dataflow.DataflowUtils.createJobName;
+import static com.google.common.truth.Truth.assertThat;
+
+import avro.shaded.com.google.common.collect.Lists;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.api.gax.paging.Page;
+import com.google.cloud.Timestamp;
+import com.google.cloud.bigquery.TableResult;
+import com.google.cloud.bigtable.admin.v2.models.StorageType;
+import com.google.cloud.bigtable.data.v2.models.RowMutation;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.Storage.BlobListOption;
+import com.google.cloud.storage.StorageOptions;
+import com.google.cloud.teleport.it.TemplateTestBase;
+import com.google.cloud.teleport.it.bigquery.DefaultBigQueryResourceManager;
+import com.google.cloud.teleport.it.bigtable.BigtableResourceManagerCluster;
+import com.google.cloud.teleport.it.bigtable.BigtableTableSpec;
+import com.google.cloud.teleport.it.bigtable.StaticBigtableResourceManager;
+import com.google.cloud.teleport.it.common.ExceptionMessageUtils;
+import com.google.cloud.teleport.it.dataflow.DataflowClient.JobInfo;
+import com.google.cloud.teleport.it.dataflow.DataflowClient.JobState;
+import com.google.cloud.teleport.it.dataflow.DataflowClient.LaunchConfig;
+import com.google.cloud.teleport.it.dataflow.DataflowOperator;
+import com.google.cloud.teleport.it.dataflow.DataflowOperator.Config;
+import com.google.cloud.teleport.it.dataflow.DataflowOperator.Result;
+import com.google.cloud.teleport.metadata.TemplateIntegrationTest;
+import com.google.cloud.teleport.v2.templates.bigtablechangestreamstobigquery.model.ChangelogColumn;
+import java.io.IOException;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.List;
+import java.util.UUID;
+import java.util.function.Supplier;
+import org.apache.commons.lang3.StringUtils;
+import org.jetbrains.annotations.NotNull;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Integration test for {@link BigtableChangeStreamsToBigQuery}.
+ */
+@Category(TemplateIntegrationTest.class)
+@TemplateIntegrationTest(BigtableChangeStreamsToBigQuery.class)
+@RunWith(JUnit4.class)
+public final class BigtableChangeStreamsToBQIT extends TemplateTestBase {
```

Comment:
nit for naming consistency:
```suggestion
public final class BigtableChangeStreamsToBigQueryIT extends TemplateTestBase {
```

Reply:
Alright!